config/spotbugs/baseline.xml

```diff
@@ -516,16 +516,16 @@
   </BugInstance>
   <BugInstance type="DM_BOXED_PRIMITIVE_FOR_PARSING" priority="1" rank="16" abbrev="Bx" category="PERFORMANCE" instanceHash="8c11cd79131cd2081e9b1222ee6040bb" instanceOccurrenceNum="0" instanceOccurrenceMax="0">
     <ShortMessage>Boxing/unboxing to parse a primitive</ShortMessage>
-    <LongMessage>Boxing/unboxing to parse a primitive com.microsoft.identity.common.internal.cache.BrokerOAuth2TokenCache.setSingleSignOnState(String, MicrosoftAccount, MicrosoftRefreshToken)</LongMessage>
```

Comment:
Let's undo/remove changes here and try to fix spotbugs issue if possible.

Reply:
Yup I'm aware of that. Normally I would remove the change in this file and try to resolve the issues manually (since they should be small enough, and it's unlikely that we'll ever come back and fix this).

Reply:
(If it's too much, i still think that it's better putting an exemption in the code itself, with a verification text. No one looks in this baseline file.)

Reply:
Heh.  I just fixed them all.